istioctl/pkg/authz/listener.go

```diff
@@ -125,7 +125,15 @@ func ParseListener(listener *v2.Listener) *ParsedListener {
 	}
 
 	for _, fc := range listener.FilterChains {
-		parsedFC := &filterChain{tlsContext: fc.TlsContext}
+		tlsContext := &auth.DownstreamTlsContext{}
+		if fc.TransportSocket != nil && fc.TransportSocket.Name == "tls" {
```

Comment:
Don't we need to check proxy version? Will panic when it runs for a older version proxy

Reply:
I am not sure what you mean? This is istioctl, we check if it has transport socket and use it if it does, otherwise use tlscontext? Why would we check the version?

Reply:
the transport socket exists for quite a long while (before istio 1.0), so I don't think you need check anyway.